bzip2.rs

```diff
@@ -1513,12 +1513,13 @@ unsafe fn uncompress(name: Option<String>) {
             outStr = stdout;
             if inStr.is_null() {
                 eprintln!(
-                    "{}: Can't open input file {}:{}.",
+                    "{}: Can't open input file {}: {}.",
                     get_program_name().display(),
                     in_name.display(),
                     display_last_os_error(),
                 );
                 if !inStr.is_null() {
+                    // this is unreachable, but in the original C source code
```

Comment:
Truncated sentence?

Reply:
no that branch is literally in the C source code, I suspect it's some copy/paste error?

https://github.com/trifectatechfoundation/libbzip2-rs/blob/9115464a466cca21f688c15f4e84b299ffa55538/bzip2.c#L1263C32-L1269C12

Reply:
Did you mean something like this?

```suggestion
                    // this is unreachable, but it exists in the original C source code
```

Reply:
yes. Though we could decide to remove it entirely of course ...